src/Files.App/UserControls/Widgets/QuickAccessWidget.xaml.cs

```diff
@@ -116,6 +122,13 @@ public QuickAccessWidget()
 
 			Loaded += QuickAccessWidget_Loaded;
 			Unloaded += QuickAccessWidget_Unloaded;
+
+			OpenInNewTabCommand = new RelayCommand<FolderCardItem>(OpenInNewTab);
+			OpenInNewWindowCommand = new RelayCommand<FolderCardItem>(OpenInNewWindow);
+			OpenInNewPaneCommand = new RelayCommand<FolderCardItem>(OpenInNewPane);
+			OpenPropertiesCommand = new RelayCommand<FolderCardItem>(OpenProperties);
+			PinToFavoritesCommand = new RelayCommand<FolderCardItem>(PinToFavorites);
+			UnpinFromFavoritesCommand = new RelayCommand<FolderCardItem>(UnpinFromFavorites);
```

Comment:
I notice these commands are used in the other widgets as well, can we try to reuse them?